webapp/components/admin_console/storage_settings.jsx

```diff
@@ -98,7 +98,7 @@ export default class StorageSettings extends AdminSettings {
                     label={
                         <FormattedMessage
                             id='admin.image.localTitle'
-                            defaultMessage='Local Directory Location:'
+                            defaultMessage='Local File Directory:'
```

Comment:
How about "Local storage directory"? 

Also, help text is wrong here, we store files and images here, not just images. 

![image](https://cloud.githubusercontent.com/assets/177788/16301797/6107af3e-3915-11e6-937f-426323192ea6.png)